src/encoded/schemas/basecalling.json

```diff
@@ -0,0 +1,104 @@
+{
+    "title": "Basecalling",
+    "$id": "/profiles/basecalling.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "required": [
+        "gpu",
+        "model",
+        "modification_tags",
+        "submission_centers",
+        "submitted_id",
+        "title",
+        "version"
+    ],
+    "identifyingProperties": [
+        "accession",
+        "submitted_id",
+        "uuid"
+    ],
+    "additionalProperties": false,
+    "mixinProperties": [
+        {
+            "$ref": "mixins.json#/accession"
+        },
+        {
+            "$ref": "mixins.json#/attribution"
+        },
+        {
+            "$ref": "mixins.json#/modified"
+        },
+        {
+            "$ref": "mixins.json#/schema_version"
+        },
+        {
+            "$ref": "mixins.json#/status"
+        },
+        {
+            "$ref": "mixins.json#/submitted"
+        },
+        {
+            "$ref": "mixins.json#/submitted_id"
+        },
+        {
+            "$ref": "mixins.json#/tags"
+        },
+        {
+            "$ref": "mixins.json#/title"
+        },
+        {
+            "$ref": "mixins.json#/uuid"
+        },
+        {
+            "$ref": "mixins.json#/version"
+        }
+    ],
+    "properties": {
+        "accession": {
+            "accessionType": "BC"
+        },
+        "schema_version": {
+            "default": "1"
+        },
+        "submitted_id": {
+            "pattern": "^[A-Z0-9]{3,}_BASECALLING_[A-Z0-9-_.]{4,}$"
+        },
+        "title": {
+            "enum": [
+                "Dorado",
+                "Guppy"
+            ]
+        },
+        "version": {
+            "pattern": "^[0-9]+\\.[0-9]+\\.[0-9]+$"
+        },
+        "gpu": {
+            "title": "GPU",
+            "description": "GPU used for basecalling",
+            "type": "string"
+        },
```

Comment:
You might consider making this suggested_enum at least so we don't need to reconcile differences in how these are entered